src/database/services/HookService.ts

```diff
@@ -0,0 +1,62 @@
+import { Hook } from '../models'
+
+interface HookConfig {
+    position: number[],
+    current_panel_id: number,
+    next_panel_set_id: number
+}
+
+/**
+ * Perform queries on the 'hooks' table
+ */
+class HookService {
```

Comment:
Do we truly need these `Service`s? What are they doing for us that a file with plain exported functions doesn't?

Reply:
That is a question for Arthur, I think it is just for organizational purposes right now but we could change it if needed